ui/lib/kmip/addon/engine.js

```diff
@@ -5,15 +5,14 @@
 
 import Engine from 'ember-engines/engine';
 import loadInitializers from 'ember-load-initializers';
-import Resolver from './resolver';
+import Resolver from 'ember-resolver';
 import config from './config/environment';
 
 const { modulePrefix } = config;
-/* eslint-disable ember/avoid-leaking-state-in-ember-objects */
-const Eng = Engine.extend({
-  modulePrefix,
-  Resolver,
-  dependencies: {
+export default class KmipEngine extends Engine {
```

Comment:
Not necessary, but I thought updating this syntax was low lift and quick win removing leaky state potential